mappings/net/minecraft/server/command/AbstractServerCommandSource.mapping

```diff
@@ -1,6 +1,6 @@
 CLASS net/minecraft/class_8839 net/minecraft/server/command/AbstractServerCommandSource
-	METHOD method_54307 withResultStorer (Lnet/minecraft/class_8935;)Lnet/minecraft/class_8839;
-		ARG 1 resultStorer
+	METHOD method_54307 withReturnValueConsumers (Lnet/minecraft/class_8935;)Lnet/minecraft/class_8839;
```

Comment:
shouldn't be plural i don't think... typo?

Reply:
```suggestion
	METHOD method_54307 withReturnValueConsumer (Lnet/minecraft/class_8935;)Lnet/minecraft/class_8839;
```